extensions/dub-link-shortener/fetch-polyfill.patch

```diff
@@ -0,0 +1,81 @@
+diff --git a/extensions/dub-link-shortener/package-lock.json b/extensions/dub-link-shortener/package-lock.json
```

Comment:
Is this file necessary?

Reply:
@steven-tey I had the same question, it looked like a git patch that was already applied. Shall we clean this up?

Reply:
Same question here 😄

Reply:
@disintegrator can we remove this patch file since it's already been applied? or do we still need to keep it?

Reply:
OK removing the patch!